convert2rhel/actions/__init__.py

```diff
@@ -567,34 +636,35 @@ def find_actions_of_severity(results, severity):
         # matched_actions will contain all actions which were skipped
         # or failed while running.
     """
-    matched_actions = [action for action in results.items() if action[1]["status"] >= STATUS_CODE[severity]]
-
+    print("find_actions_of_severity")
+    matched_actions = [message for message in results.items() if key(message) >= STATUS_CODE[severity]]
+    print(matched_actions)
```

Comment:
No prints statement needed


```suggestion
    matched_actions = [message for message in results.items() if key(message) >= STATUS_CODE[severity]]
```

Reply:
Will remove these thanks